Gemfile

```diff
@@ -0,0 +1,9 @@
+source 'https://rubygems.org'
+gemspec
+
+gem 'bzip2-ruby', :github => 'chewi/bzip2-ruby', :branch => 'master'
```

Comment:
Sadly, official gem supports only ruby 1.9.3 :cry: 
Scaptimony has internal engine for bzip2.
I can use `bunzip2 ...` as bunzip is used only once